### PR TITLE
added a health check to the docker-compose.yml fil

### PR DESCRIPTION
**Pull request by carson-may**

### CODE REVIEW

**Comment by abarcomb-work** [comment]
The PR should have been made to the ensf400-ws24 branch, not to main.

**Comment by abarcomb-work** [comment]
Pull requests should contain useful information about the reasoning behind the changes for the repository maintainer to enable them to prioritize PRs. This is part of the collaborative workflow.

File: docker-compose.yml [hunk 1]
```diff
@@ -18,6 +18,12 @@ services:
       - ${PWD}:/app
       - /var/run/docker.sock:/var/run/docker.sock
     network_mode: "host"
+    healthcheck:
```
Comment (growlf) [inline docker-compose.yml:21]: This will always fail due to no web service running on localhost (as defined in the same compose)

**Comment by growlf** [comment]
A good first attempt at contributing.  Please adjust the target branch and ensure that you code works before submitting a PR.

**Pull request closed without merging**